js/src/extension-renderer.ts

```diff
@@ -14,6 +14,4 @@ if (window.require) {
 }
```

Comment:
The TypeScript transpiler complains on this file that `window.require` does not exist.